src/System.Net.WebHeaderCollection/tests/WebHeaderCollectionTest.cs

```diff
@@ -529,6 +529,108 @@ public void Add_InvalidHeader_ThrowsArgumentException(string header, string para
             AssertExtensions.Throws<ArgumentException>(paramName, () => headers.Add(header));
         }
 
+        private string headerType = "Set-Cookie";
```

Comment:
These fields can all be `const` and then the first char of their names should be upper case.